webrtc-stats.html

```diff
@@ -1064,6 +1075,99 @@
           </section>
         </div>
       </section>
+      <section id="contributingsourcestats-dict*">
+        <h3>
+          <dfn>RTCRTPContributingSourceStats</dfn> dictionary
```

Comment:
Actually they don't; we have RTCRTPStreamStats, RTCInboundRTPStreamStats and RTCOutboundRTPStreamStats.